File: pycript/encryption.py
from . import encoding, decoding
from .execution import execute_command
from .gethelpers import string_to_bytes, bytes_to_string

#Jsonvalueencrypt     -->   Parameterencrypt 

def Parameterencrypt(selectedlang, path, data,headers_str):
    body_parameter_byte = str(list(string_to_bytes(data)))
    result = execute_command(selectedlang, path, body_parameter_byte,headers_str)

    if result is not False:
        body, header = result
        string_body = bytes_to_string(body)
        return string_body,header
    else:
        return data,headers_str

File: src/main/java/com/pycript/EncDec/Encryption.java
package com.pycript.EncDec;

import org.apache.commons.lang3.tuple.Pair;

public class Encryption {

    public static Pair<byte[], String> Parameterencrypt(String selectedLang, String path, byte[] data, String headersStr) {
        Pair<byte[], String> result = Execution.executeCommand(selectedLang, path, data, headersStr);

        if (result != null) {
            return result;
        } else {
            return org.apache.commons.lang3.tuple.ImmutablePair.of(data, headersStr);
        }
    }
}
